routes/users.js

```diff
@@ -29,4 +29,7 @@ router.patch("/rejectDiff", authenticate, authorizeRoles([SUPERUSER]), users.rej
 router.patch("/:userId", authenticate, authorizeRoles([SUPERUSER]), users.updateUser);
 router.get("/suggestedUsers/:skillId", authenticate, authorizeRoles([SUPERUSER]), users.getSuggestedUsers);
 
+// WARNING!! - One time Script/Route to do migration.
+router.post("/api/migrate", authenticate, authorizeRoles([SUPERUSER]), users.migrate);
```

Comment:
The URL will look like `api.realdevsquad.com/api/migrate`. How does this sound?
Are we migrating the whole API?

Reply:
I'll change it to api.realdevsquad.com**/users/migrate**